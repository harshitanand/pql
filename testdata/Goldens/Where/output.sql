SELECT * FROM "StormEvents" WHERE ("DamageProperty" > 5000) AND (coalesce("EventType" = 'Thunderstorm Wind', FALSE));
