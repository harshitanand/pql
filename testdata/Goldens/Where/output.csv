MISSISSIPPI,Thunderstorm Wind,20000
